pymc3/backends/smc_text.py

```diff
@@ -196,8 +196,14 @@ def load_atmip_params(self, stage_number):
         else:
             prev = stage_number - 1
         pm._log.info('Loading parameters from completed stage {}'.format(prev))
-        with open(self.atmip_path(prev), 'rb') as buff:
-            return pickle.load(buff)
+
+        with model:
```

Comment:
I like that this nesting is explicit, but just for interest's sake:

starting with python 2.7 you can use multiple context managers in one line
```
with open(...) as buff, model:
    step = pickle.load(buff)
```